lib/Api/PickupPlaceApi.php

```diff
@@ -10,11 +10,11 @@
  */
 
 /**
- * Foxdeli Pickup Place Service API
+ * Foxdeli Customer Service API
  *
- * Foxdeli API documentation
+ * Foxdeli API implements JWT tokens as its chosen method of authentication, requiring that these tokens be included in the HTTP header X-API-Key for each request.
```

Comment:
Tohle je nejaka hloupost, ta hlavicka, takhle urcite byt nema?

Ty jo ted na to koukam a ono je asi rozbity tohle kompozitni schéma 🤦 https://api.foxdeli.com/schema/foxdeli-api.swagger.json

Ještě že si toho @jbesta všiml... 

Každopádně přegeneruj to prosím z toho schématu jak to bylo. A já se pokusim opravit to kompozitní schéma v dokumentaci, protože on to měl bejt multiservice a on to udělal sice jako multiservice ale hodil to popisem jak kdyby pod jedno schéma 🙏 

Zbytek se zdá v pohodě.